packages/react-menu/src/components/MenuItem/MenuItem.types.ts

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { ComponentProps, ShorthandProps, ObjectSlotProp } from '@fluentui/react-utils';
+import { RefObjectFunction } from '@fluentui/react-hooks';
+
+export interface MenuItemProps extends ComponentProps, React.HTMLAttributes<HTMLElement> {
+  /**
+   * Icon slot rendered before children content
+   */
+  icon?: ShorthandProps;
+}
+
+export interface MenuItemState extends MenuItemProps {
+  /**
+   * Ref to the root slot
+   */
+  ref: RefObjectFunction<HTMLElement>;
```

Comment:
```suggestion
  ref: React.MutableRefObject<HTMLElement>;
```

https://github.com/microsoft/fluentui/pull/16865/files#r572082937